sklearn/hpsearch/README.md

```diff
@@ -0,0 +1,34 @@
+# Parallelized Hyperparameter Search with Google Container Engine
+
+## Introduction
+
+This sample package helps you run `scikit-learn`'s `GridSearchCV` and `RandomizedSearchCV` on [Google Container Engine](https://cloud.google.com/container-engine/).
```

Comment:
Links to `scikit-learn`, `GridSearchCV`, `RandomizedSearchCV` would be good here.

Reply:
done